configs/ppyoloe/README.md

```diff
@@ -0,0 +1,123 @@
+English | [简体中文](README_cn.md)
+
+# PP-YOLOE
+
+## Table of Contents
+- [Introduction](#Introduction)
+- [Model Zoo](#Model-Zoo)
+- [Getting Start](#Getting-Start)
+- [Appendix](#Appendix)
+
+## Introduction
+PP-YOLOE is an excellent single-stage anchor-free model based on PP-YOLOv2, surpassing a variety of popular yolo model. PP-YOLOE has a series of models, named s/m/l/x, which are configured through width multiplier and depth multiplier. PP-YOLOE avoids using special operators, such as DCN or matrix nms, to be deployed friendly on various hardware. For more details, please refer to our report.
```

Comment:
a variety of popular yolo model -> a variety of popular yolo models

Reply:
done, thx!